include/core/tag.h

```diff
@@ -75,6 +75,21 @@ typedef struct {
     microstep_t microstep;
 } tag_t;
 
+////////////////  External Functions
+
+// Bring clock synchronization adjustment into scope. These functions are
+// defined in clock-sync.c
+#if defined(_LF_CLOCK_SYNC_ON)
+extern void clock_sync_apply_offset(instant_t *t);
+extern void clock_sync_remove_offset(instant_t *t);
+extern void clock_sync_set_constant_bias(interval_t offset);
+#else
+// Without clock synchronization enabled we can just optimize these away.
+#define clock_sync_apply_offset(x)
+#define clock_sync_remove_offset(x)
+#define clock_sync_set_constant_bias(x)
+#endif
```

Comment:
I don't see why we wouldn't allow these functions even if clock sync is not on.  Could allow the user to adjust the clock sync.

Reply:
See final comment